src/framework/json_ld.cr

```diff
@@ -104,7 +104,7 @@ module Ktistec
     end
 
     private def self.expand_value(term, value, context, loader)
-      if term.in?(["https://www.w3.org/ns/activitystreams#content", "https://www.w3.org/ns/activitystreams#name", "https://www.w3.org/ns/activitystreams#summary"])
+      if term.in?(["https://www.w3.org/ns/activitystreams#content", "https://www.w3.org/ns/activitystreams#name", "https://www.w3.org/ns/activitystreams#summary", "https://www.w3.org/ns/activitystreams#attachment"])
```

Comment:
@toddsundsted I'm not 100% sure with this change. It required changing how `ActivityPub::Object` mapped from `ld_json` to the properties. I will note this below in comments. But this was necessary to get the attachments to work for `Actor` AFAICT.

Reply:
this shouldn't be necessary once some additional definitions are added to the @context.